src/sotoki/constants.py

```diff
@@ -189,9 +189,20 @@ def any_restriction(self):
     def __post_init__(self):
         self.dump_domain = self.domain  # dumps are named after unfixed domains
         self.domain = FIXED_DOMAINS.get(self.domain, self.domain)
-        self.iso_lang_1, self.iso_lang_3 = lang_for_domain(self.domain)
+        self.iso_lang_1, self.iso_lang_3, is_multi_language = lang_for_domain(
```

Comment:
Here you should probably change the whole prototype to return lists instead of adding an additional parameter.